src/main/java/com/cleanroommc/groovyscript/gameobjects/GohMetaMethod.java

```diff
@@ -0,0 +1,54 @@
+package com.cleanroommc.groovyscript.gameobjects;
+
+import com.cleanroommc.groovyscript.sandbox.expand.IDocumented;
+import groovy.lang.MetaMethod;
+import org.codehaus.groovy.reflection.CachedClass;
+import org.codehaus.groovy.reflection.ReflectionCache;
+
+import java.lang.reflect.Modifier;
+
+public class GohMetaMethod extends MetaMethod implements IDocumented {
+
+    private final GameObjectHandler<?> closure;
+    private final Class<?> owner;
+    private final String documentation;
+
+    GohMetaMethod(GameObjectHandler<?> closure, Class<?>[] nativeParamTypes, Class<?> owner, String documentation) {
```

Comment:
pretty sure that, since you pass in the GOH, you dont need to pass in documentation